doc/source/timedeltas.rst

```diff
@@ -283,6 +283,18 @@ Rounded division (floor-division) of a ``timedelta64[ns]`` Series by a scalar
    td // pd.Timedelta(days=3, hours=4)
    pd.Timedelta(days=3, hours=4) // td
 
```

Comment:
add a ref tag here

Reply:
You mean a "(:issue:`19365`)"?

Reply:
no a section reference

Reply:
Do you mean something like ".. _timedeltas.divmod:" after line 297?